code/modules/networks/computer3/mainframe2/misc_terms.dm

```diff
@@ -2749,7 +2749,6 @@ TYPEINFO(/obj/machinery/networked/printer)
 	var/obj/beam/next
 	var/limit = 48
 
-	dir = 2
 	layer = NOLIGHT_EFFECTS_LAYER_BASE
```

Comment:
?

Reply:
Also the other instances of this change in the file

Reply:
ah right
so the pr started out as something being done by hand
i ditched the dir = 2 in multiple objects because the default dir for `obj` is south anyway and it was an unneeded line.
I can put it back if you like but it works fine without.